src/engine/enginemaster.cpp

```diff
@@ -45,7 +45,7 @@ EngineMaster::EngineMaster(UserSettingsPointer pConfig,
           m_masterHandle(registerChannelGroup(group)),
           m_headphoneHandle(registerChannelGroup("[Headphone]")),
           m_masterOutputHandle(registerChannelGroup("[MasterOutput]")),
-          m_busMicrophonesHandle(registerChannelGroup("[Microphones]")),
+          m_busMicrophonesHandle(registerChannelGroup("[MicrophoneMix]")),
```

Comment:
We can put any channel to microphones bus using the [ChannelN], talkover switch. 
So this effect Deck is not exclusive to Microphones as the name suggests. 
We have also the takoverDucking CO  so I think it is better to call this [BusTalkover]